src/modules/dashboard/lists/SupplyAssetsList/SupplyAssetsList.tsx

```diff
@@ -89,6 +88,7 @@ export const SupplyAssetsList = () => {
         return [
           {
             ...reserve,
+            reserve,
```

Comment:
Right, so basically the values from `reserve` are depended on for finding out these caps and usage.  And these warnings/tooltips/status components can be deeply nested in a component hierarchy, very far from the original page components where this reserve data is available.  So rather than passing through the several reserve fields needed through as _multiple_ props all the way through to the nested component(s) that need them to show the warning, its cleaner to pass through just the single reserve.  Now, for the dashboard lists, most of the reserve fields are passed through as individual props when they're spread with `{...item}`.  However, not _all_ fields were passed in, and rather than ensuring multiple fields were plumbed through correctly, and/or creating a new representation in TypeScript, it was a path of least resistance to pass in `reserve` is it's own singular prop and allow the utility function to just work with a fully represented `ComputedReserveData` type.

This isn't ideal.  This is redundant, since most _but not all_ reserve fields are passed through to the components, but not all fields are continuously passed through.  For example, `SupplyAssetsListItem` may have access to all the reserve fields, but `ListItemWrapper` doesn't, as it's more general and has a more restrictive props list.

I think the next step to reduce this plumbing and redundancy of passing through the reserve entity would be to create a React context to hold this info and wrap the page level with it.  Then the deeply nested component can just access the context rather than a plumbed through prop.